src/sonic-config-engine/portconfig.py

```diff
@@ -376,7 +378,8 @@ def get_config(self):
                     'alias': self._breakout_capabilities[alias_id],
                     'lanes': ','.join(lanes),
                     'speed': str(entry.default_speed),
-                    'index': self._indexes[lane_id]
+                    'index': self._indexes[lane_id],
+                    'subport': "0" if total_num_ports == 1 else str(alias_id + 1)
```

Comment:
So this field will be populated even if the hwsku.json doesn't define any subport?